vss_layers/README.md

```diff
@@ -0,0 +1,243 @@
+such information cannot be defined as part of the common model or catalog --
+it simply differs too often between usage situations.  Sometimes it is
+required to view the system's data model with different layer configurations
+even within a single implementation.  Therefore, a layered approach is needed
+in the creation of a standard data model and data catalog.
+
+The fact that this may differ even on instances of vehicles of the same make
+shows that the ability to add/remove layers dynamically might be needed in the
+technology stack implementation.  Another consideration is for layers to be
+individually updated during software updates.
+
+### Access control
+
+A frequent augmentation of the data model information is definition of access
+permission rules.  These boil down to specific signals that a specific actor
+is allowed or not allowed to access.  Once again, this differs depending on
+the situation and should be defined as a separate layer.
+
+## General definition of VSS-layers
+
+The layer concept definition should impose very little restrictions.
+Conventions and particular tools may introduce their own restrictions
+for certain usage, at a later time.
+
+The layer concept itself is able to do all of the following:
+
+1. Modify existing metadata and ultimately redefine any signal (i.e. change
+   its metadata) compared to a previous definition for example the definition
+   in the standard catalog.
+1. Add new signals. It can be done in the private/ branch as recommended but the
+   mechanism allows adding them anywhere.
+1. Add new metadata, or relationships to new concepts for the purpose of
+   creating deployment-models, bindings to existing technology, classification
+   and many other things.
+1. Remove signals from the final data model (SYNTAX TO BE DEFINED)
+
+VSS-layers can also define new concepts as described in the following
+scenarios:
+
+## VSS-layer definition variants
+
+There are two obvious approaches to add new information to a data model like
+VSS.
+
+#### Standard Layer
+
+The first option is to list the signal first and add metadata below
+it. This is straight forward since it mirrors the core VSS definition.
+
+Let's call that a `Standard Layer`.
+
+Here we see an arbitrary example of adding the known accuracy of a particular
+measurement in this particular system (likely to be unique per vehicle model
+due to the sensors involved), and defining which ECU in the electrical
+architecture is responsible to produce the original data for this signal.
+
+```
+Vehicle.Chassis.Wheel2.Brake.Fluidlevel:
+   accuracy:  -1% - +2%
+   ECU:  BrakeManagementUnit
+```
+
+#### Reverse Layer
+
+The second way is to flip the definition around.  First define the new
+concept, and below it list metadata (that is specific to the concept).
+The definitions of the metadata is is likely to refer back to the VSS signal
+names.
+
+Let's call that a `Reverse Layer`.
+
```

Comment:
I think this section "breaks" the model.

I think it is "legal", and "possible", because it is akin to just extending private data (i.e. creating new branches/subtrees) and filling whatever data you like.

But in this specific example I would assume it is bad practice, so we should not endorse it.